app/views/user_mailer/added_to_collection_notification.html.erb

```diff
@@ -1,19 +1,14 @@
 <% content_for :message do %>
-  <p>Dear <%= style_bold(@user.default_pseud.byline) %>,</p>
+  <p><%= t(".greeting", name: style_bold(@user.default_pseud.byline)).html_safe %></p>
 
-  <p>The collection maintainers of <%= style_link(@collection.title, collection_url(@collection)) %> have
-    added your work (<%= style_link(@work.title.html_safe, work_url(@work)) %>) to their collection!</p>
+  <p><%= t(".part1", collection: style_link(@collection.title, collection_url(@collection)).html_safe, work: style_link(@work.title.html_safe, work_url(@work)).html_safe).html_safe %></p>
```

Comment:
Can you rename the keys? We're going to be updating all the keys that are like `part1` or `para1` to have more descriptive names ([AO3-6240](https://otwarchive.atlassian.net/browse/AO3-6240)), and it would be better not to add new ones to the pile.

Reply:
Note to self, this still needs doing.